### PR TITLE
update change password box

### PR DESCRIPTION
**Pull request by win11905**

### CODE REVIEW

File: app/views/users/index.html.haml [hunk 1]
```diff
@@ -5,31 +5,32 @@
...
     -#%td.uinfo= in_place_editor_field :user, 'email_for_editing', {}, :rows => 1
   %tr
-    %th.uinfo Password
-    %td.uinfo
+    %th Password
+    %td
       = form_tag :action => 'chg_passwd', :method => 'post' do
         %table
           %tr
-            %td= password_field_tag 'passwd'
```
Comment (wytesk133) [inline app/views/users/index.html.haml]: Why not `%td= password_field_tag 'passwd', nil, class: 'form-control'`?

File: app/views/users/index.html.haml [hunk 2]
```diff
@@ -5,31 +5,32 @@
...
+            %td
+              %input{:type => "password", :class => "form-control", :name => "passwd", :id => "passwd"}
             %td (new)
           %tr
-            %td= password_field_tag 'passwd_verify'
```
Comment (wytesk133) [inline app/views/users/index.html.haml]: Also `%td= password_field_tag 'passwd_verify', nil, class: 'form-control'`

File: app/views/users/index.html.haml [hunk 3]
```diff
@@ -5,31 +5,32 @@
...
+            %td
+              %input{:type => "password", :class => "form-control", :name => "passwd_verify", :id => "passwd_verify"}
             %td (verify)
           %tr
             %td{:colspan => "2"}
-              = submit_tag 'change password'
```
Comment (wytesk133) [inline app/views/users/index.html.haml]: `= submit_tag 'change password', class: 'btn btn-default'`

**Comment by nattee** [comment]
Thank you and sorry that it took so long

**Pull request merged**